src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+
+    public HashUtil() {
+    }
+
+    public static byte[] getSalt() {
+        SecureRandom secureRandom = new SecureRandom();
+        byte[] salt = new byte[16];
```

Comment:
i dunno, but if it occured once it's not good idea to create constant variable for that (OPTIONALLY)
```suggestion
        byte[] salt = new byte[DEFAULT_SIZE];
```